python/natural-lang/function.yaml.template

```diff
@@ -0,0 +1,20 @@
+metadata:
```

Comment:
1. You set env twice, once as MANIPULATION_KIND with the value from the template and once as TO_LANG with hardcoded language
2. Please use UpperCase for values: targetLang -> TargetLang